ui/public/diagnoseReport.html

```diff
@@ -10,6 +10,8 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="%PUBLIC_URL%/supportedBrowsers.js"></script>
```

Comment:
Can we ensure that these scripts will run before our main scripts?

Reply:
Yes, according to the result I observed, I will do some more tests to confirm.